src/components/Croupier.tsx

```diff
@@ -100,7 +100,15 @@ const Croupier = () => {
           />
           <PlayerTitle
             gamePieceAppearance={uiState.deckPositions[`${player.uid}PlayerDeck`]}
-            offset={[0, 10, 0]}
+            offset={
+              // TODO: use positioner for titles as well
+              [
+                [0, 10, 0],
+                [0, 10, 0],
+                [0, -13, 0],
```

Comment:
Why does only one of the players have a different offset? Previously the player titles were positioned relative to a GamePieceGroup, making all of this unnecessary

Reply:
For the third player we want the name to be placed below deck, not above.

It happens for **only** the third one because it's not rotated like second and forth. Mathematically speaking the first one is also rotated with angle = 0.